### DISCUSSION TITLE
local ip?

### DISCUSSION THREAD

**Discussion by F1qa** [bedrock-tool/bedrocktool#28] [Q&A] [answered]
I Have tried to download some worlds. however I am unsure what to put as the Local IP.

My computers local ipv4 IP does not do anything when put into the ip box? says unable to connect

Would like some clarification on how to use this tool.

Thanks.

**Comment by olebeck** [comment, marked as answer]
Local ip is 127.0.0.1, due to windows things you may need to follow this https://doc.pmmp.io/en/rtfd/faq/connecting/win10localhostcantconnect.html
